packages/flutter_rearch/lib/src/side_effects.dart

```diff
@@ -25,6 +25,25 @@ extension BuiltinWidgetSideEffects on WidgetSideEffectRegistrar {
   TextEditingController textEditingController({String? initialText}) =>
       _textEditingController(this, initialText: initialText);
 
+  /// Provides a way to easily get a copy of a [ScrollController].
+  ScrollController scrollController({
+    double initialScrollOffset = 0.0,
+    bool keepScrollOffset = true,
+    String? debugLabel,
+    void Function(ScrollPosition)? onAttach,
+    void Function(ScrollPosition)? onDetach,
+  }) =>
+      use.disposable(
```

Comment:
was wondering whether this should specify the keys/dependencies of `use.disposable`, but based on the fact that ScrollController fields are all final, and appear to be for initial values, I'm not sure it's necessary.

I know this PR is pretty trivial, but would you mind adding a test or two in `packages/flutter_rearch/test/side_effects_test.dart`? Just to ensure basic functionality.

Reply:
Well, seems attach/detach pretty different to dispose